corehq/apps/reports/models.py

```diff
@@ -713,7 +716,11 @@ def _get_and_send_report(self, language, emails):
         from corehq.apps.reports.views import get_scheduled_report_response
 
         with localize(language):
-            title = _("Scheduled report from CommCare HQ")
+            title = (
+                _(DEFAULT_REPORT_NOTIF_SUBJECT)
+                if self.email_subject == DEFAULT_REPORT_NOTIF_SUBJECT
+                else self.email_subject
+            )
```

Comment:
does the default subject get properly migrated on reads of old models that don't have it?